metadata.json

```diff
@@ -102,12 +103,8 @@
   ],
   "requirements": [
     {
-      "name": "pe",
-      "version_requirement": ">= 3.7.0 < 2015.4.0"
-    },
-    {
       "name": "puppet",
-      "version_requirement": ">= 3.0.0 < 5.0.0"
+      "version_requirement": ">= 4.5.0 < 5.0.0"
```

Comment:
+1 bugfixes in the type system and data in modules make 4.5.x a solid recommendation